src/MoviePage/MovieCard/MovieCard.js

```diff
@@ -16,7 +16,7 @@ export default function MovieCard({movieDetails}) {
 
         <div className="box runtime-container">
           <h1 className="title rating-title">Rating</h1>
-          <p className="value average-rating">{movieDetails.average_rating}</p>
+          <p className="value average-rating">{movieDetails.average_rating.toFixed(1)}</p>
```

Comment:
What does `.toFixed(1)` do again?

Reply:
it shortens or extends  a number to its first decimal.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed